nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/AbstractTrashService.java

```diff
@@ -306,33 +307,49 @@ public void untrashDocuments(List<DocumentModel> docs) {
      */
     protected static final Pattern COLLISION_PATTERN = Pattern.compile("(.*)\\.[0-9]{13,}");
```

Comment:
these fields are constants, move them at the beginning of the classe and add `@since 11.1`

Reply:
This is not introduced either. But it can be moved in a cleaning